ci/deploy.yaml

```diff
@@ -0,0 +1,31 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: "alpine"
+
+params:
+  PRIVATE_KEY: 
```

Comment:
Where will this be provided?

Reply:
in the pipeline config, fetched from Vault